queue.h

```diff
@@ -57,6 +57,26 @@ queue_item_destroy(queue_item_t *);
 void
 queue_item_init(queue_item_t *, void *);
 
+/*
+ * Create a new queue_t object.
+ *
+ * This will create and initialize a new queue_t and return a handle to that
+ * object.
```

Comment:
Same here. Mention that `NULL` will be returned if the queue could not be created.

Reply:
Done.